examples/diagram-generator/index.html

```diff
@@ -5,19 +5,17 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>QuickERD</title>
 
-    <link rel="stylesheet" type="text/css" href="libs/rappid.css">
+    <!-- Assuming you put the rappid.min.css file in this path -->
+    <link rel="stylesheet" type="text/css" href="libs/rappid.min.css">
 
     <link rel="stylesheet" href="../../dist/quick-erd.css">
 
     <script src="https://cdn.jsdelivr.net/npm/jquery@3.7.1/dist/jquery.min.js" integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo=" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.21/lodash.min.js" integrity="sha256-qXBd/EfAdjOA2FGrGAG+b3YBn2tn5A6bhz+LSgYD96k=" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/backbone@1.5.0/backbone-min.min.js"></script>
     
+    <!-- Assuming you put the rappid.min.js file in this path -->
     <script src="libs/rappid.min.js"></script>
-    <script src="libs/rappid.min.js"></script>
-
-    <!-- <script src="../../dist/quick-erd.umd.cjs"></script>
```

Comment:
These comments were there on purpose so that developers could know how to use the library with the UMD build

Reply:
I will return the commented line